src/Components/TagPickerGridComponent/TagPickerGridComponent/TagPicker.tsx

```diff
@@ -9,103 +8,103 @@ import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
 initializeIcons();
 
 export interface ITagPickerProps {
-  labelText?: string,
-  tagDisplayName?: string,
-  selectedItems?: ITag[],
-  onChange?: (items?: ITag[]) => void;
-  onEmptyInputFocus?: (selectedItems?: ITag[]) => Promise<ITag[]>;
-	onResolveSuggestions?: (filter: string, selectedItems?: ITag[]) => Promise<ITag[]>;
+    inputLabel?:string;
+    noResultsFoundLabel?: string;
+    removeButtonLabel?: string;
+
+    labelText?: string;
```

Comment:
we could move this into a sub property to end up with something like this.props.labels.noresultsfound

Reply:
The label properties have been moved into a sub-property (lables) of type ITagPickerLabelProps.